.travis.yml

```diff
@@ -1,35 +1,60 @@
 ---
 language: node_js
 node_js:
-  - '6'
-  - '4'
+  # we recommend testing addons with the same minimum supported node version as Ember CLI
+  # so that your addon works for all apps
+  - "10"
 
-sudo: false
+dist: trusty
+
+addons:
+  chrome: stable
 
 cache:
   directories:
-    - node_modules
+    - $HOME/.npm
 
 env:
-  - EMBER_TRY_SCENARIO=default
-  - EMBER_TRY_SCENARIO=ember-release
-  - EMBER_TRY_SCENARIO=ember-beta
-  - EMBER_TRY_SCENARIO=ember-canary
+  global:
+    # See https://git.io/vdao3 for details.
+    - JOBS=1
+
+branches:
+  only:
+    - master
+    # npm version tags
+    - /^v\d+\.\d+\.\d+/
 
-matrix:
+jobs:
   fast_finish: true
   allow_failures:
     - env: EMBER_TRY_SCENARIO=ember-canary
 
-before_install:
-  - export PATH=/usr/local/phantomjs-2.0.0/bin:$PATH
-  - "npm config set spin false"
-  - "npm install -g npm@^2"
+  include:
+    # runs linting and tests with current locked deps
+    - stage: "Tests"
+      name: "Tests"
+      script:
+        - npm run lint:hbs
+        - npm run lint:js
+        - npm test
 
-install:
-  - npm install -g bower
-  - npm install
-  - bower install
+    - stage: "Additional Tests"
+      name: "Floating Dependencies"
+      install:
+        - npm install --no-package-lock
+      script:
+        - npm test
+
+    # we recommend new addons test the current and previous LTS
+    # as well as latest stable release (bonus points to beta/canary)
+    - env: EMBER_TRY_SCENARIO=ember-lts-3.12
```

Comment:
Should we add 3.8 && even 3.4 for the time being?  It is such a large bump and likely if users haven't explored different solution with autoprefixer, they might be on older version?

Reply:
I've added 3.8.  3.4 doesn't work, `vendor.css` ends up empty.

I'm not sure it's reasonable to expect compatibility with 3.4.  Could we make this a 1.x release and state that we support 3.8 and above?  Folks on current versions can continue to use and folks that want updates (and are using 3.4) may need to do some work.

Reply:
Yep that is totally fine!  Was just ensuring we support the earliest versions that we could!  Thanks a ton!